README.md

```diff
@@ -68,96 +64,128 @@ you may receive this error:
 
 Answer ``yes``.
 
-## Configuration and Start-up
+## Running edX devstack locally _(optional, but recommended)_
 
-Create a user who has permission to view the course:
+Micromasters can work with a live instance of edX, but it's recommended that
+you get it running locally. It's obviously more configurable that way, and you'll
+likely need to run it locally for other projects in the future.
 
-    docker-compose run web python manage.py createsuperuser
+#### 1) Install edX
+Run through the
+[instructions provided by edX](https://openedx.atlassian.net/wiki/display/OpenOPS/Running+Devstack)
```

Comment:
I should have mentioned this to you earlier, but I think https://edx-installing-configuring-and-running.readthedocs.io/en/latest/installation/devstack/install_devstack.html may be more authoritative than confluence.